chart/validator/templates/cleanup.yaml

```diff
@@ -86,23 +86,32 @@ metadata:
     "helm.sh/hook": pre-delete
 spec:
   template:
+    metadata:
+      labels:
+        app: validator-cleanup-job
     spec:
       restartPolicy: Never
       serviceAccountName: spectro-cleanup
       containers:
       - name: validator-cleanup
-        image: gcr.io/spectro-images-public/release/spectro-cleanup:1.1.0
+        image: gcr.io/spectro-images-public/release/spectro-cleanup:1.2.0
         command: ["/cleanup"]
         env:
         - name: CLEANUP_DELAY_SECONDS
-          value: {{ mul 10 (max 1 (len (required ".Values.plugins is required!" .Values.plugins))) | quote }}
+          value: {{ mul 15 (max 1 (len (required ".Values.plugins is required!" .Values.plugins))) | quote }}
+        {{- if .Values.cleanup.grpcServerEnabled }}
+        - name: CLEANUP_GRPC_SERVER_ENABLED
+          value: "true"
+        - name: CLEANUP_GRPC_SERVER_PORT
+          value: {{ required ".Values.cleanup.port is required!" .Values.cleanup.port | toString | quote }}
+        {{- end }}
         resources:
           requests:
             cpu: "10m"
-            memory: "25Mi"
+            memory: "50Mi"
           limits:
             cpu: "100m"
-            memory: "50Mi"
+            memory: "100Mi"
```

Comment:
Throughout my testing, spectro-cleanup was occasionally `OOMKilled` so i bumped the memory allocated to the job.